starter-code/styles/modules.css

```diff
@@ -66,3 +71,25 @@
          - The main-nav should be all on one row, with padding spaced better
            for a larger screen.
 */
+/*DONE*/
+@media screen and (min-width: 1000px) {
+  .icon-menu {
+    display: none;
+    width: 800px;
```

Comment:
Again, when thinking responsive, think proportional units. So, for width, use percentage over pixels.